VideoLocker/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -294,8 +285,7 @@ protected void onOnline() {
     @Override
     protected void onOffline() {
         super.onOffline();
-        showErrorMessage(getString(R.string.no_connectivity),
-                getString(R.string.network_not_connected), false);
+        showErrorMessage(getString(R.string.no_connectivity), getString(R.string.network_not_connected));
```

Comment:
Whatever the decision, lets make it official by updating this wiki: https://openedx.atlassian.net/wiki/display/MA/Offline+Errors